diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -372,13 +372,14 @@ controller_interface::CallbackReturn DiffDriveController::on_configure(
       tf_prefix = std::string(get_node()->get_namespace());
     }
 
-    if (tf_prefix == "/")
+    // Make sure prefix does not start with '/' and always ends with '/'
+    if (tf_prefix.back() != '/')
     {
-      tf_prefix = "";
+      tf_prefix = tf_prefix + "/";
     }
-    else
+    if (tf_prefix.front() == '/')
     {
-      tf_prefix = tf_prefix + "/";
+      tf_prefix.erase(0,1);
```

Comment:
```suggestion
      tf_prefix.erase(0, 1);
```

Reply:
@kjvanteeffelen it seems that you have disabled that maintainers can make changes to your PR. Please fix the clang issue reported with pre-commit, see the [contributing guidelines](https://github.com/ros-controls/ros2_control/blob/master/CONTRIBUTING.md#contributing-via-pull-requests).